examples/gpu/Makefile

```diff
@@ -20,8 +20,8 @@ override FB_IMG := $(abspath ${FB_IMG})
 override BUILD_DIR := $(abspath ${BUILD_DIR})
```

Comment:
This line repeats line 14 above, this time with `override`.  `abspath` is needed only once; `export` should come after the `override` if you want the changed variable to be in the environment for subprocesses.